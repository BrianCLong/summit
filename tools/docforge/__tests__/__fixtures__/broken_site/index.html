<!DOCTYPE html>
<html>
  <body>
    <a href="missing/page.html">Missing Page</a>
  </body>
</html>
