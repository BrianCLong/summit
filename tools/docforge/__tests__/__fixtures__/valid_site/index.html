<!doctype html>
<html>
  <body>
    <a href="docs/page.html">Docs</a>
    <a href="https://example.com">External</a>
  </body>
</html>
