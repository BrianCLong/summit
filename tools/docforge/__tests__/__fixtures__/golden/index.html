<!DOCTYPE html>
<html lang="en">
<head>
  <meta charset="utf-8" />
  <title>DocForge Documentation</title>
  <meta name="viewport" content="width=device-width, initial-scale=1" />
  <style>
    :root { font-family: system-ui, sans-serif; color: #1a1a1a; background: #f9fafb; }
    body { margin: 0 auto; max-width: 960px; padding: 2rem; line-height: 1.6; }
    header { border-bottom: 1px solid #e5e7eb; margin-bottom: 2rem; padding-bottom: 1rem; }
    nav { font-size: 0.9rem; color: #4b5563; margin-bottom: 0.5rem; }
    h1 { font-size: 2rem; margin: 0 0 1rem; }
    h2 { font-size: 1.5rem; margin-top: 1.5rem; }
    table { width: 100%; border-collapse: collapse; margin-top: 1rem; }
    th, td { border: 1px solid #e5e7eb; padding: 0.5rem; text-align: left; vertical-align: top; }
    th { background: #f3f4f6; }
    article { background: #ffffff; border: 1px solid #e5e7eb; border-radius: 8px; padding: 1rem; margin-bottom: 1.5rem; box-shadow: 0 1px 2px rgba(15, 23, 42, 0.08); }
    code { background: #f3f4f6; padding: 0.1rem 0.3rem; border-radius: 4px; font-size: 0.95rem; }
    ul { padding-left: 1.25rem; }
    a { color: #2563eb; text-decoration: none; }
    a:hover { text-decoration: underline; }
    footer { margin-top: 3rem; font-size: 0.85rem; color: #6b7280; }
  </style>
</head>
<body>
  <header>
    <nav><span>Home</span></nav>
    <h1>DocForge Documentation</h1>
    
  </header>
  <main>
    <section>
<h2>Available Versions</h2>
<ul><li><a href="./2.3.4/index.html">2.3.4</a></li></ul>
</section>
<section id="modules">
<h2>API Modules</h2>
<table>
<thead><tr><th>Module</th><th>Language</th><th>Summary</th></tr></thead>
<tbody><tr>
        <td><a href="./2.3.4/modules/pcbo__engine_go.html">pcbo/engine.go</a></td>
        <td>Go</td>
        <td>Planner orchestrates policy evaluation and scoring logic.</td>
      </tr>
<tr>
        <td><a href="./2.3.4/modules/src__analytics_js.html">src/analytics.js</a></td>
        <td>JavaScript</td>
        <td>Calculate the moving average for a numeric series.</td>
      </tr>
<tr>
        <td><a href="./2.3.4/modules/tools__report_py.html">tools/report.py</a></td>
        <td>Python</td>
        <td>Reporting helpers for DocForge sample fixtures.</td>
      </tr></tbody>
</table>
</section>
<section id="adrs">
<h2>Architecture Decision Records</h2>
<table>
<thead><tr><th>ADR</th><th>Summary</th></tr></thead>
<tbody><tr>
        <td><a href="./2.3.4/adrs/adr-0001-adopt-docforge.html">ADR 0001: Adopt DocForge</a></td>
        <td>- Status: Accepted</td>
      </tr></tbody>
</table>
</section>
  </main>
  <footer>
    Generated by DocForge.
  </footer>
</body>
</html>